src/state/app-store.tsx

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from "redux";
+import { appReducer } from "./app.reducer";
+import thunkMiddleware from "redux-thunk";
+import { composeWithDevTools } from "redux-devtools-extension";
+
+const composedEnhancer = composeWithDevTools(applyMiddleware(thunkMiddleware));
```

Comment:
@pectom, usunąłem to
```
type WindowWithDevTools = Window & {
  __REDUX_DEVTOOLS_EXTENSION__: () => StoreEnhancer<unknown, {}>;
};

const isReduxDevtoolsExtAvailable = (
  arg: Window | WindowWithDevTools
): arg is WindowWithDevTools => {
  return "__REDUX_DEVTOOLS_EXTENSION__" in arg;
};

```
ponieważ nie potrzebowałem dodać `thunkMiddleware` i najprostszy sposób na to wygląda tak jak jest teraz. 
Nie mniej jednak nie sprawdzałem czy `composeWithDevTools` działa tak samo jak to co pisałeś ty, bo skupiłem się na interfejsie ( sprawdzę to jutro). Więc jeżeli to jest źle rozwiązanie - to mów od razu :D

Reply:
Wszystko jest w porządku.